package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vtex/react-csv-parse",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
That's a major version bump since the API changed in a non-backwards compatible way.

Reply:
True, i'd have done that afterwards but it makes sense to do it now.